components/atom/progressBar/src/settings.js

```diff
@@ -7,11 +7,17 @@ export const TYPES = {
 }
 
 export const SIZES = {
+  XL: 'xl',
```

Comment:
```suggestion
  EXTRA_LARGE: 'extraLarge',
```

in order to be consistent with the existing naming?